app/shared/src/errors/__tests__/__snapshots__/PartialFailure.test.js.snap

```diff
@@ -61,7 +61,7 @@ Array [
           ]
         }
       >
-        Some parts of the page may be missing due to partial server error.
```

Comment:
@tbergq I don't understand this change. Can you please explain it? (fix it)

Reply:
`Translation` renders first with `this.state.translatedString = ''` then async calls native, gets the response and calls `this.setState({translatedString})`

So on first render, translation just renders empty string, that is why ^^ has changed

Reply:
In this case, I am afraid I cannot accept this PR since we are not able to test simple pass-through type of translation... 😞 

This is for me obviously failing test.

Reply:
> This is for me obviously failing test

This is really another component, mocking it should solve this

Reply:
I see it's not pass through translation in this case... :-/